erc20-bridge-token/contracts/BridgeToken.sol

```diff
@@ -0,0 +1,68 @@
+pragma solidity ^0.8;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+
+
+contract BridgeToken is ERC20 {
+
+    address private _controller; // has CONTROL role
+
+    string private _name;
+    string private _symbol;
+    uint8 private _decimals;
+
+    uint64 _metadataLastUpdated;
+
+    constructor(
+        string memory name_, 
+        string memory symbol_,
+        uint8 decimals_
+    ) ERC20(name_, symbol_ ) {
+        _controller = msg.sender;
```

Comment:
Warning: After upgradability is implemented (both for token and factory) this should still work; so this should be pointing to the proxy.

Reply:
After a short discussion with Marcel, we agreed we could ignore this comment. 
https://github.com/aurora-is-near/rainbow-token-connector/pull/99/commits/2fc2366f96992ffcd20c67ba666624bc475bb0be